### PR TITLE
SIDM-7885 eJudiciary supporting notification

### PR DESCRIPTION
**Pull request by nikola-naydenov-hmcts**
Came out pretty subtle

image

### CODE REVIEW

File: src/main/views/user-details.njk [hunk 1]
```diff
@@ -19,7 +20,19 @@
   ]) }}
 {% endblock %}
 
+{% set eJudiciaryNotification %}
+  <p class="govuk-notification-banner__heading">
+    Please check with the eJudiciary support team to see if there are related accounts.
```
Comment (jburke-idam) [inline src/main/views/user-details.njk:25]: I guess if we want to change this text we'd just do a release? (It's not paramaterized)

**Review by jburke-idam — APPROVED** [review]

File: src/main/views/user-details.njk (hunk 1 above)
Comment (nikola-naydenov-hmcts) [reply src/main/views/user-details.njk:25]: Yeah the project is not internationalised, all the text is hard coded. I would have to redo a fair amount of the project for this to be reasonably manageable in a different way other than this.

**Review by kremi — CHANGES_REQUESTED** [review]
Can we check the correct value for ssoProvider, please?

File: src/main/views/user-details.njk [hunk 2]
```diff
@@ -19,7 +20,19 @@
...
+  </p>
+{% endset %}
+
 {% block content %}
+  {% if content.user.ssoProvider and content.user.ssoProvider.toLowerCase().includes('ejudiciary') %}
```
[the remark this answers is not in the document]
Comment (nikola-naydenov-hmcts) [reply src/main/views/user-details.njk]: yeah, you're right. its actually azure 😨

File: src/main/views/user-details.njk (hunk 1 above)
Comment (nikola-naydenov-hmcts) [reply src/main/views/user-details.njk:25]: I have a plan

**Review by kremi — APPROVED** [review]

**Pull request merged**